pkgs/development/web/nodejs/use-correct-env-in-tests-v22.patch

```diff
@@ -0,0 +1,24 @@
+`/usr/bin/env` is not available.
+
+--- old/test/common/assertSnapshot.js
++++ new/test/common/assertSnapshot.js
+@@ -81,2 +81,2 @@ async function spawnAndAssert(filename, transform = (x) => x, { tty = false, ...
+-  const executable = tty ? path.join(__dirname, '../..', 'tools/pseudo-tty.py') : process.execPath;
+-  const args = tty ? [process.execPath, ...flags, filename] : [...flags, filename];
++  const executable = tty ? 'python3' : process.execPath;
```

Comment:
```js
const python = process.env.PYTHON || (common.isWindows ? 'python' : 'python3')
```
E.g. see https://github.com/nodejs/node/blob/c0c7e866dc2bed108290a514fdd50db3c886c84f/test/parallel/test-child-process-set-blocking.js#L28

Also, would it make sense to upstream this entire patch?

Reply:
The reason we need this patch is because `/usr/bin/env` is not available on the sandbox, I don't know if it makes sense to upstream this – although `process.env.PYTHON` bit might get approval.

Reply:
Note that `isWindows` is not necessary because of https://github.com/nodejs/node/blob/87598d4b63ef2c827a2bebdfa0f1540c35718519/test/common/assertSnapshot.js#L76-L79

Reply:
I've opened https://github.com/nodejs/node/pull/53938